### DISCUSSION TITLE
How to use SQL `IN` operator in PRQL filter?

### DISCUSSION THREAD

**Discussion by strokirk** [PRQL/prql#4326] [Q&A] [answered]
I tried finding a way to express the `IN` operator in PRQL, but so far haven't figured it out, and can't find it in the examples. This was my first try:

```
from db.tracks
filter artist | in ("Bob Marley", "Bob Dylan")
```

Is the `IN` operator supported yet?

**Comment by eitsupi** [comment, marked as answer]
Example of the `in` function is here.
https://eitsupi.github.io/querying-with-prql/indexing#filtering-rows

```elm
from tab
filter (IATA_CODE_Reporting_Airline | in ["AA", "DL"])
```
